src/features/user/MFV2/TpoProjects/index.tsx

```diff
@@ -77,7 +77,6 @@ export default function ProjectsContainer({ profile }: Props) {
                   >
                     <ProjectSnippet
                       project={project.properties}
```

Comment:
Badge translations do not work here, as the `projectDetails` translations are not included for this page.